rotate_array.py

```diff
@@ -0,0 +1,18 @@
+def rotate_array(arr, k):
+
+    n = len(arr)
```

Comment:
give meaningful name to variable

Reply:
Required changes are made, Kindly review it.